src/Pages/Review/components/Basket/Basket.jsx

```diff
@@ -0,0 +1,41 @@
+  import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import omit from 'lodash/omit';
+import BasketCard from '../BasketCard/BasketCard.jsx';
+import './Basket.scss';
+
+const mapStateToProps = (state) => Object({
+    basket: state.basket,
+});
+
+export const BasketComponent = (props) => {
+    const basket = omit(props, ['basket']);
+    return (
+        <div>
+            <h1>Your Basket</h1>
+            <p>You're currently looking at purchasing these tickets:</p>
+            <div>
+                { props.basket.map((item, index) => {
+                    if (item && item.quantity) {
+                        return (
+                            <BasketCard
+                                item={item}
+                                key={`basket-item-${index}`}
+                            />
+                        );
+                    }
+                    return '';
```

Comment:
you can return `null` or `undefined` here also but i guess its a matter of opinion. Whatever we do we should stick to it throughout the codebase.